### PR TITLE
Added changing to the directory of the FtcRobotController folder

### PR DESCRIPTION
**Pull request by cbroosky**
When the workspace root folder is not FtcRobotController, ./gradlew fails because the file does not exist. So I edited run.js to change the directory to the correct folder in order to allow for different workspace roots. 

It uses the file that is currently open in the editor, grabs the uri, then it truncates it to the correct folder. I only tested on Ubuntu 20.04, I assume it will work in Windows as well. I also added code that opens the app through adb. I guess if adb is not installed it won't work.

TL;DR
Features Added:
- directory change to correct directory (FtcRobotController) when building the app
- opens app after build through adb

Potential Problems:
- if adb is not installed 2nd part of run script will fail
- not tested on Windows

### CODE REVIEW

**Comment by kidsonfilms-python-rules** [comment]
# PR #4 Accepted
Your PR has been reviewed and has been accepted into `master` branch of this project

Your PR fits the following criteria:
- [x] bug-free
- [x] Malware/Security Vulnerabilities not found
- [x] Tested on multiple platforms [Tested on Windows 10 x64 and MacOS Big Sur. PR author claims to test on Ubuntu 20.04]
- [ ] Has no points of potential failure

## Has No Points of Potential Failure
An issue will be created once this PR is merged to resolve the `adb` failure point
